zombienet/tests/0007-full_node_warp_sync.toml

```diff
@@ -31,12 +31,30 @@ cumulus_based = true
   validator = false
```

Comment:
Is there a reason why dave/eve/ferdie are not validators?

Reply:
I used the existing `full_node_catching_up` as the basis of the test


https://github.com/paritytech/cumulus/blob/master/zombienet/tests/0003-full_node_catching_up.toml#L31-L39